backend/api/Controllers/ReportController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Api.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+[ApiController]
+[Route("reports")]
+public class ReportController : ControllerBase
+{
+    private readonly ILogger<ReportController> _logger;
+    private readonly ReportService _reportService;
+
+    public ReportController(ILogger<ReportController> logger, ReportService reportService)
+    {
+        _logger = logger;
```

Comment:
Not in use. See similar comment for event controller.

Reply:
Fixed